client/state/sites/features/reducer.js

```diff
@@ -46,6 +70,13 @@ export function featuresReducer( state = {}, { type, siteId, features, error } )
 				error: error,
 				isRequesting: false,
 			} );
+		case JETPACK_SITES_FEATURES_FETCH:
+			return { ...state, isRequestingJetpackSitesFeatures: true };
+		case JETPACK_SITES_FEATURES_FETCH_SUCCESS:
+		case JETPACK_SITES_FEATURES_FETCH_FAILURE:
+			return { ...state, isRequestingJetpackSitesFeatures: false };
+		case JETPACK_SITES_FEATURES_RECEIVE:
+			return updateBulkFeatures( state, features );
```

Comment:
It's a bummer we don't have unit tests to cover this reducer. Can we take care of it before shipping this one? We could do it in this PR, of course, but alternatively, we could create tests for the existing reducer in another PR, and then amend/update them in this one. Whatever you prefer!

Reply:
Yeah I agree they would be good. I made a card for it in Asana: 1202619025189113-as-1203519690464537

I already have another PR in the works which is optimizing some of the plugins selectors, maybe I will add these there or in their own PR. I'd like to go ahead and ship this one when possible.